examples/consumer.py

```diff
@@ -102,6 +102,9 @@ def print_assignment(consumer, partitions):
                                  (msg.topic(), msg.partition(), msg.offset(),
                                   str(msg.key())))
                 print(msg.value())
+                # Store the offset associated with msg to a local cache. Stored offsets are committed to Kafka by a background thread every AutoCommitIntervalMs
+                # Explicitly storing offsets after processing gives at-least once semantics, the default behavior does not.
+                c.store_offsets(message=msg)
```

Comment:
Not sure message=msg is helpful, it suggests this is not the default usage and that there are other parameters, which there aren't. Just pass the msg as is without named parameters.

Reply:
Makes sense, updated now.